src/test/java/htsjdk/samtools/CRAMIteratorTest.java

```diff
@@ -0,0 +1,50 @@
+package htsjdk.samtools;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.cram.ref.ReferenceSource;
+import htsjdk.samtools.seekablestream.SeekableStream;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+
+/**
+ * @author Anton_Mazur@epam.com, EPAM Systems, Inc.
+ *
+ * This test serve for verifying CRAMIterator records validation using strict validation strategy
+ **/
+
+public class CRAMIteratorTest extends HtsjdkTest {
+    private static final File refFile = new File("src/test/resources/htsjdk/samtools/cram/ce.fa");
+    private static final File cramFile = new File("src/test/resources/htsjdk/samtools/cram/ce#supp.3.0.cram");
+    ReferenceSource source = new ReferenceSource(refFile);
```

Comment:
We don't publish a style guide, but for new code we try to use "final" everywhere possible (paramaters, locals, etc.).